package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "ember-axe",
-  "version": "0.0.2",
+  "version": "1.0.0",
```

Comment:
Should be fine as a patch level, but we generally bump the version separate from every other change. Let's not make this change in this PR. 😄